README.md

```diff
@@ -44,7 +44,7 @@ export default {
 };
 ```
 
-You must pass a `src` property with the link of the image:
+You must pass an `src` property with the link of the image:
```

Comment:
I think in this case is "a" right? Since "sec" and "property" start by consonant